doc/internal/man3/OSSL_METHOD_STORE.pod

```diff
@@ -88,24 +88,24 @@ and the ones passed to the ossl_method_store_free().
 =head2 Cache Functions
 
 ossl_method_store_cache_get() queries the cache associated with the B<store>
-for an method identified by B<nid> that matches the property query
+for a method identified by B<nid> that matches the property query
 B<prop_query>.
 The result, if any, is returned in B<method>.
 
 ossl_method_store_cache_set() sets a cache entry identified by B<nid> with the
 property query B<prop_query> in the B<store>.
-Future cache gets will return the specified B<method>.
+Future calls to ossl_method_store_cache_get will return the specified B<method>.
```

Comment:
I suggest to change `ossl_method_store_cache_get` to `ossl_method_store_cache_get()`

Reply:
Good point

Reply:
Ok will address it in the new PR created from the right account.

Reply:
Fixed in #9806 that's also created under right account.